Security/Web_Security.mediawiki

```diff
@@ -0,0 +1,614 @@
+<div style="max-width: 75em;">
+<table>
+  <tr>
+    <td style="white-space: nowrap;">__TOC__</td>
+    <td style="vertical-align: top; padding-left: 1em;">The goal of this document is to help operational teams with creating secure web applications. All Mozilla sites and deployments are expected to follow the recommendations below.
+
+The Enterprise Information Security (EIS) team maintains this document as a reference guide to navigate the rapidly changing landscape of web security. Changes are reviewed and merged by the OpSec team, and broadcast to the various Operational teams.
+
+Updates to this page should be submitted to the [https://github.com/mozilla/wikimo_opsec source repository on github].
+    </td>
+  </tr>
+</table>
+
+= Transport Layer Security =
+
+== HTTPS ==
+
+Transport Layer Security (TLS/SSL) provides assurances about the confidentiality, authentication, and integrity of all communications both inside and outside of Mozilla. To protect our users and networked systems, all systems must use encrypted TLS connections for their network communications.  Support and use of TLS is mandatory for all new systems, and existing systems are expected to add support for exclusive TLS as soon as possible.
```

Comment:
Repetition, could shorten to:
To protect our users and networked systems, support and use of TLS is mandatory for all systems.

Rationale: systems that do not currently support TLS are simply not compliant. The standard/guideline represents a known-good-state that we know we can and must (or should) achieve.
Note that "must" and "mandatory" imply there is no exception (else its a "should")